common/utils/hostname.js

```diff
@@ -2,8 +2,8 @@
  * Created by elad.benedict on 9/24/2015.
  */
 
-const os = require('os');
-const child_process = require('child_process');
+var os = require('os');
```

Comment:
You're overriding Guy's fix from https://github.com/kaltura/liveDVR/pull/486, why?